tests/test_acl.py

```diff
@@ -70,22 +96,70 @@ def test_AclTableCreation(self, dvs):
                     assert True
                 else:
                     assert False
-    
+
         assert set(member_groups) == set(acl_table_groups)
-    
-        # check port binding 
+
+    def verify_acl_port_binding(self, dvs, adb, expt, bind_ports):
+        atbl = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE_GROUP")
+        acl_table_groups = atbl.getKeys()
+        assert len(acl_table_groups) == expt
+
         atbl = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_PORT")
-    
         port_groups = []
         for p in [dvs.asicdb.portnamemap[portname] for portname in bind_ports]:
             (status, fvs) = atbl.get(p)
             for fv in fvs:
                 if fv[0] == "SAI_PORT_ATTR_INGRESS_ACL":
                     assert fv[1] in acl_table_groups
                     port_groups.append(fv[1])
-    
+
+        assert len(port_groups) == expt
         assert set(port_groups) == set(acl_table_groups)
-    
+
+    def verify_acl_lag_binding(self, adb, expt):
+        atbl = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE_GROUP")
+        acl_table_groups = atbl.getKeys()
+        assert len(acl_table_groups) == expt
+
+        if expt == 0:
+            return
+
+        port_groups = []
+        atbl_lag = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_LAG")
+        lag_keys = atbl_lag.getKeys();
+        for lag_key in lag_keys:
+            (status, lagfvs) = atbl_lag.get(lag_key)
+            for lagfv in lagfvs:
+                if lagfv[0] == "SAI_LAG_ATTR_INGRESS_ACL":
+                    assert lagfv[1] in acl_table_groups
+                    port_groups.append(lagfv[1])
+
+        assert len(port_groups) == expt
+        assert set(port_groups) == set(acl_table_groups)
+
+    def test_AclTableCreation(self, dvs):
+        db = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+        adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+
+        # create ACL_TABLE in config db
+        bind_ports = ["Ethernet0", "Ethernet4"]
+        tbl = swsscommon.Table(db, "ACL_TABLE")
+        fvs = swsscommon.FieldValuePairs([("policy_desc", "test"), ("type", "L3"), ("ports", ",".join(bind_ports))])
+        tbl.set("test", fvs)
+        time.sleep(1)
+
+        # check acl table in asic db
+        test_acl_table_id = self.get_acl_table_id(dvs, adb)
+
+        # check acl table group in asic db
+        self.verify_acl_group(adb, 2)
+
+        # check acl table group member
+        self.verify_acl_group_member(adb, 2, test_acl_table_id)
```

Comment:
>verify_acl_group_member [](start = 13, length = 23)

this function signature should be 

verify_acl_group_member(adb, [acl_group_ids], acl_table_id)

it should verify the acl_groups have corresponding acl_group_member which has the acl_table_id.